apps/web/src/components/Icon/Icon.stories.tsx

```diff
@@ -0,0 +1,46 @@
+import Icon from './Icon';
+import { ReactComponent as RedditIcon } from '../../assets/redditIcon.svg';
+import { ReactComponent as UserIcon } from '../../assets/userIcon.svg';
+import { ReactComponent as HomeIcon } from '../../assets/homeIcon.svg';
+
+export default {
+  title: 'Components/Icon',
+  component: Icon,
+};
+
+interface IconArgs {
+  /**
+   * child icon SVG component passed to the Icon component
+   */
+  children: JSX.Element;
+  /**
+   * global css styles applied to all of our Icon components
+   */
+  style?: React.CSSProperties;
+}
+
+const sharedStyles = {
```

Comment:
Why are you passing this as a prop to all instances of the `Icon` component? Can't it just live in that scope and save you from passing a prop?